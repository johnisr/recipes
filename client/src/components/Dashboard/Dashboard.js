import React from 'react';
import { Link } from 'react-router-dom';
import { Button, Icon } from 'semantic-ui-react';

const Dashboard = () => {
  return (
    <div>
      Dashboard!
      <div className="fixed-action-btn">
        <Button
          icon
          positive
          circular
          as={Link}
          to="/recipesNew"
          floated="right"
        >
          <Icon name="add" size="large" />
        </Button>
      </div>
    </div>
  );
};

export default Dashboard;
