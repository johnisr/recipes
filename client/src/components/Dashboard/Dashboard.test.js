import React from 'react';
import { shallow } from 'enzyme';
import { Dashboard } from './Dashboard';

test('should render correctly', () => {
  const wrapper = shallow(<Dashboard user recipes={[]} />);
  expect(wrapper).toMatchSnapshot();
});
