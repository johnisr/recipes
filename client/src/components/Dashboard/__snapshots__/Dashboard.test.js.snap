// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Dashboard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "Dashboard!",
        <div
          className="fixed-action-btn"
>
          <Button
                    as={[Function]}
                    circular={true}
                    floated="right"
                    icon={true}
                    positive={true}
                    role="button"
                    to="/recipesNew"
          >
                    <Icon
                              as="i"
                              name="add"
                              size="large"
                    />
          </Button>
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      "Dashboard!",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Button
            as={[Function]}
            circular={true}
            floated="right"
            icon={true}
            positive={true}
            role="button"
            to="/recipesNew"
>
            <Icon
                        as="i"
                        name="add"
                        size="large"
            />
</Button>,
          "className": "fixed-action-btn",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "as": [Function],
            "children": <Icon
              as="i"
              name="add"
              size="large"
/>,
            "circular": true,
            "floated": "right",
            "icon": true,
            "positive": true,
            "role": "button",
            "to": "/recipesNew",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "as": "i",
              "name": "add",
              "size": "large",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "Dashboard!",
          <div
            className="fixed-action-btn"
>
            <Button
                        as={[Function]}
                        circular={true}
                        floated="right"
                        icon={true}
                        positive={true}
                        role="button"
                        to="/recipesNew"
            >
                        <Icon
                                    as="i"
                                    name="add"
                                    size="large"
                        />
            </Button>
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        "Dashboard!",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Button
              as={[Function]}
              circular={true}
              floated="right"
              icon={true}
              positive={true}
              role="button"
              to="/recipesNew"
>
              <Icon
                            as="i"
                            name="add"
                            size="large"
              />
</Button>,
            "className": "fixed-action-btn",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "as": [Function],
              "children": <Icon
                as="i"
                name="add"
                size="large"
/>,
              "circular": true,
              "floated": "right",
              "icon": true,
              "positive": true,
              "role": "button",
              "to": "/recipesNew",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "as": "i",
                "name": "add",
                "size": "large",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
